.github/workflows/tox.yml

```diff
@@ -117,22 +118,25 @@ jobs:
         include:
         - tox_env: py36
           os: ubuntu-20.04
-          python-version: 3.6
+          python-version: "3.6"
```

Comment:
Please don't add extra quotes where they are unnecessary.

Reply:
In this case I think that adding them is highly recommended because I already made the mistake of writing `3.10` which is converted to `3.1`. The correct value for this field is a string, not a float, so using quotes would be highly recommended.

This avoids future mistakes when someone would want to bump the version and they endup writing a float with zero at the end. Being consistent is better here.

PS. Obviously that I would not advocate for adding quotes to things that are obvious strings.